tests/functional/test_upload_service.py

```diff
@@ -138,6 +164,7 @@ def _upload_area_record(self):
         return db.query(DbUploadArea).filter(DbUploadArea.uuid == self.upload_area_uuid).one_or_none()
 
     def _checksum_record(self, filename):
+        # TODO do we need to do this this way around
```

Comment:
I'm not sure I understand this comment

Reply:
Ah my bad.  Actually PyCharm's bad - it usually flags all your TODOs and has you review them.   The comment is a note to me that I meant to take care of before I pushed.  Basically this function retrieves a file record so it can retrieve a checksum record, and the function that is calling it retrieves a file record after that.  I was speculating about reworking that.  But I think not.  I will remove the comment.